spec/departure_comparator_spec.rb

```diff
@@ -68,27 +68,11 @@
   describe 'compare' do
     context 'with scheduled routes' do
       it 'checks if every route is present and next departure has correct scheduled time' do
-        Bojangles.stub(:get_avail_departure_times!) do
```

Comment:
... I didn't even test the compare method! I don't know what I was thinking. Wow.